### PR TITLE
update: next solver project goal

### PR DESCRIPTION
**Pull request by lcnr**
cc @nikomatsakis

### CODE REVIEW

File: src/2024h2/next-solver.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Next-generation trait solver
+
+| Metadata | |
+| --- | --- |
+| Owner(s) | [@lcnr] |
+| Teams | [Types] |
+| Status | WIP |
```
Comment (compiler-errors) [inline src/2024h2/next-solver.md:7]: In progress?

Comment (lcnr) [reply src/2024h2/next-solver.md:7]: idk, it's what's used in the `impl-trait-everywhere` doc :thinking: :shrug:

Comment (compiler-errors) [reply src/2024h2/next-solver.md:7]: OH lol i think i misread this as "TODO fill in status" not "literally in progress"

**Comment by lqd** [comment]
this https://github.com/rust-lang/rust-project-goals/blob/main/src/2024h2/candidates.md?plain=1#L85 will link to the old file

**Comment by nikomatsakis** [comment]
I'll merge this later on!

**Comment by lcnr** [comment]
> this https://github.com/rust-lang/rust-project-goals/blob/main/src/2024h2/candidates.md?plain=1#L85 will link to the old file

fixed

**Pull request merged**